optlang/glpk_interface.py

```diff
@@ -579,6 +579,9 @@ def __setstate__(self, repr_dict):
         if repr_dict['glpk_status'] == 'optimal':
             self.optimize()  # since the start is an optimal solution, nothing will happen here
 
```

Comment:
That would probably work, but perhaps a better solution would be to use weakrefs for the references to Models in Variables, Constraints and Objectives?